src/engine/effects/engineeffect.h

```diff
@@ -42,8 +42,8 @@ class EngineEffect : public EffectsRequestHandler {
                  const EffectProcessor::EnableState enableState,
                  const GroupFeatureState& groupFeatures);
 
-    bool enabled() const {
-        return m_enableState != EffectProcessor::DISABLED;
```

Comment:
This was confusing because the function did not actually test whether the EngineEffect was in the enabled state, it tested that it wasn't in the fully disabled state.